ncl/ncl_entries/dewtemp.ipynb

```diff
@@ -0,0 +1,101 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# dewtemp_trh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "NCL's [`dewtemp_trh`](https://www.ncl.ucar.edu/Document/Functions/Built-in/dewtemp_trh.shtml) calculates the dew point temperature given temperature and relative humidity using the equations from John Dutton's _\"Ceaseless Wind\"_ (pg. 273-274) and returns a temperature in Kelvin.\n",
+    "\n",
+    "<div class=\"admonition alert alert-info\">\n",
+    "    <p class=\"admonition-title\" style=\"font-weight:bold\">Important Note</p>\n",
+    "    To convert from Kelvin to Celsius <code>-273.15</code> and to convert from Celsius to Kelvin <code>+273.15</code>\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Grab and Go"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Input: Single Value\n",
+    "from geocat.comp import dewtemp\n",
+    "\n",
+    "temp_c = 18  # Celsius\n",
+    "relative_humidity = 46.5  # %\n",
+    "\n",
+    "dewtemp(temp_c + 273.15, relative_humidity) - 273.15  # Returns in Celsius"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Input: List/Array\n",
+    "from geocat.comp import dewtemp\n",
+    "\n",
+    "temp_kelvin = [291.15, 274.14, 360.3, 314]  # Kelvin\n",
+    "relative_humidity = [46.5, 5, 96.5, 1]  # %\n",
+    "\n",
+    "dewtemp(temp_kelvin, relative_humidity) - 273.15  # Returns in Celsius"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "---"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Python Resources\n",
+    "- [GeoCAT-comp Documentation](https://geocat-comp.readthedocs.io/en/stable/)\n",
```

Comment:
Should this be the section for the `dewtemp` function? https://geocat-comp.readthedocs.io/en/latest/user_api/generated/geocat.comp.meteorology.dewtemp.html

Reply:
+1